spec/sidekiq/middleware/server/datadog_spec.rb

```diff
@@ -93,4 +93,16 @@
       ])
     end
   end
+
+  context 'when sidekiq/api is required' do
+    before do
+      require 'sidekiq/api'
+    end
+
+    it 'should not raise any errors' do
+      expect {
```

Comment:
this breaks rubocop, please see https://travis-ci.org/bsm/sidekiq-datadog/builds/535889673

Reply:
@dim addressed in a1eec313d3889ccddb8230f2c05410c7ddaf1a8b